integrationTests/factory/statusComponents/statusComponents_test.go

```diff
@@ -62,7 +62,6 @@ func TestStatusComponents_Create_Close_ShouldWork(t *testing.T) {
 		managedCoreComponents.NodesShuffler(),
 		managedBootstrapComponents.ShardCoordinator().SelfId(),
 		managedBootstrapComponents.EpochBootstrapParams(),
-		managedBootstrapComponents.EpochBootstrapParams().Epoch(),
```

Comment:
handle `WaitingListFixEnableEpoch` as well

Reply:
thanks, deleted the wrong line :man_facepalming: